administration/src/application-verification/VerificationController.tsx

```diff
@@ -70,8 +72,17 @@ const ApplicationVerification = ({ applicationVerificationAccessKey }: Applicati
     variables: { applicationVerificationAccessKey },
   })
 
+  // TODO handle two error messages of graphQL queries
+
   if (loading) return <CircularProgress style={{ margin: 'auto' }} />
-  else if (error || !data) return <ErrorHandler refetch={refetch} />
+  else if (error)
+    return (
+      <InvalidLink
+        title={getMessageFromApolloError(error).title}
+        description={getMessageFromApolloError(error).description}
+      />
+    )
```

Comment:
```suggestion
    const { title, description}  = getMessageFromApolloError(error)
    return <InvalidLink title={title} description={description} />
```

Also: It is weird to me that we show an InvalidLink component for any error. Couldn't there a different error?
What happens on connection / server errors? I think we should show a refetch button there?

Reply:
yes you are right about the refetch. Probably we shouldn't call it `InvalidLink` but extend the `ErrorHandler` with a optional description